flujo/application/core/executor_core.py

```
@@ -367,6 +367,26 @@ async def _bound(frame: _Any) -> _Any:
+
+            _sm_policy = _SMPolicy()
+
+            async def _sm_bound(frame: ExecutionFrame[_Any]) -> StepOutcome[StepResult]:
+                return await _sm_policy.execute(self, frame)
+
+            # Only register if not already present from framework registry
+            if self.policy_registry.get(_SM) is None:
+                self.policy_registry.register(_SM, _sm_bound)
+        except Exception:
+            # Defensive: never break core init due to optional policy wiring
+            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)